easybuild/easyconfigs/c/cisDIVERSITY/cisDIVERSITY-1.1-foss-2020a-Python-2.7.18.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'MakeCp'
+
+name = 'cisDIVERSITY'
+version = '1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/NarlikarLab/cisDIVERSITY/'
+description = """A module discovery tool used for finding diverse sequence architectures, 
+each one characterized by presence or absence of de novo motifs."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
```

Comment:
@pavelToman Can you clarify why you're using this ancient toolchain here?

Very soon that will no longer be installable (like on RHEL9)

Reply:
It needs python 2: https://github.com/vscentrum/vsc-software-stack/issues/435#issuecomment-2405207422

Reply:
Maybe the foss/2021b could be used, there is `SciPy-bundle-2021.10-foss-2021b-Python-2.7.18` with numpy for python 2. I am gonna try it